linera-views/src/views.rs

```diff
@@ -30,6 +30,9 @@ pub trait View<C>: Sized {
     /// Discards all pending changes. After that `flush` should have no effect to storage.
     fn rollback(&mut self);
 
+    /// Returns true if the flush is going to be non-trivial
```

Comment:
Optional: rephrasing a bit.

```suggestion
    /// Returns [`true`] if flushing this view would result in changes to the persistent storage.
```